### PR TITLE
Biqian/appserver

### PR DESCRIPTION
**Pull request by bjqian**

### CODE REVIEW

File: src/Pods/AppServer/Startup.cs [hunk 1]
```diff
@@ -47,5 +52,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
...
+
+        private void AddComandHandlers(string connectionString)
+        {
+            const string sender = "AppServer";
+            var crash = MessageHandler.CreateCommandHandler(Commands.General.Crash, cmd =>
+               {
+                   Environment.Exit(1);
+                   return Task.CompletedTask;
+               });
+            Task.Run(async () => await MessageClient.ConnectAsync(connectionString, sender, crash));
```
Comment (vwxyzh) [inline src/Pods/AppServer/Startup.cs]: no reference for MessageClient, gc will collect it...

Comment (bjqian) [reply src/Pods/AppServer/Startup.cs]: fixed

File: src/Pods/AppServer/Program.cs [hunk 2]
```diff
@@ -12,16 +13,28 @@ namespace Azure.SignalRBench.AppServer
...
+        {
+            var configuration = new ConfigurationBuilder()
+               .AddEnvironmentVariables()
+               .Build();
+            return Host.CreateDefaultBuilder(args)
+              .ConfigureLogging((ILoggingBuilder logging) =>
+              {
+                  logging.ClearProviders();
+                  logging.AddConsole();
+                  logging.AddProvider(new BlobLoggerProvider(configuration[StorageConnectionStringKey], "appserver", ""));
```
Comment (vwxyzh) [inline src/Pods/AppServer/Program.cs]: add pod name?

Comment (bjqian) [reply src/Pods/AppServer/Program.cs]: fixed

**Review by vwxyzh — APPROVED** [review]

**Pull request merged**